File: src/main/java/dev/slne/discord/exception/ticket/UnableToGetTicketNameException.java
package dev.slne.discord.exception.ticket;

import dev.slne.discord.exception.DiscordException;
import java.io.Serial;

public class UnableToGetTicketNameException extends DiscordException {

  @Serial
  private static final long serialVersionUID = -575479591760370835L;

  public UnableToGetTicketNameException(String message) {
    super(message);
  }

  public UnableToGetTicketNameException(String message, Throwable cause) {
    super(message, cause);
  }

  public UnableToGetTicketNameException(DiscordException exception) {
    super(exception);
  }
}

File: src/main/kotlin/dev/slne/discord/exception/ticket/UnableToGetTicketNameException.kt
package dev.slne.discord.exception.ticket

import dev.slne.discord.exception.DiscordException
import java.io.Serial

class UnableToGetTicketNameException : DiscordException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(exception: DiscordException) : super(exception)

    companion object {
        @Serial
        private val serialVersionUID = -575479591760370835L
    }
}
